Threadx Support for Drivers Examples and Test for AM62AX

jblanchard-jblopen opened the pull request:
Add ThreadX support for drivers examples and tests for the R5 and A53 single core configurations.

venkatesan-krishnamoorthy commented on examples/drivers/ddr/ddr_ecc_test_main_esm/.project/project_am62ax.js:
```diff
@@ -31,7 +31,16 @@ const libdirs_freertos = {
 };
 
-const includes_r5f = {
+const libdirs_threadx = {
```
Need to add generated file in the same commit as the example is added.

jblanchard-jblopen committed:
am62ax: drivers: Add driver examples for threadx.

- add threadx build option to each relevant example project file
- added flash configuration files
- add example.syscfg files

Fixes: SITSW-5614

Signed-off-by: Edouard Haas <edouard.haas@jblopen.com>

jblanchard-jblopen commented:
Squashed development and generated files together into one commit as requested.
